src/main/java/io/github/tetherless_world/jena_elasticsearch/ElasticsearchGraphMaker.java

```diff
@@ -143,7 +142,7 @@ public Graph openGraph(String name, boolean strict) {
             // there is no graph with this name yet
             if (strict) {
                 // throw an error instead of creating the graph
-                logger.error("Cannot open graph '{}'; does not exist", validIndexName);
+                this.logger.error("Cannot open graph '{}'; does not exist", validIndexName);
```

Comment:
Don't need this prefix.

Reply:
Is it bad practice to include the "this" ? I thought it was better to explicitly include it but I'm curious as to what is generally best

Reply:
It's not bad practice, it's just not common.